src/screens/styles.ts

```diff
@@ -4,7 +4,89 @@ export const styles = StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    alignItems: 'center',
+    // alignItems: 'center',
     backgroundColor: 'white',
+    paddingHorizontal: 15, // should use paddingHOrizontal to ensure it stretches across the screen so bg color doesn't change
+    gap: 8,
+    fontFamily: 'Poppins_400Regular',
+  },
+  page: {
+    flex: 1,
+    margin: '20%',
+  },
+  image: {
+    //flex: 1,
+    justifyContent: 'center',
+    width: '100%',
+    height: 231,
+    borderRadius: 10,
+    //paddingLeft: 15,
+  },
+  name: {
+    // flex: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  imageName: {
+    flexDirection: 'row',
+    paddingTop: 15,
+  },
+  userName: {
```

Comment:
Just a small convention thing, username is usually spelled in all lowercase because it is one word.